styles/styles.css

```diff
@@ -0,0 +1,601 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        4.3 Work
+        4.4 Services
+        5.5 Form
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --purple-light: #F2F0FF; /* floating text */
+    --grey: #A7A7A; /* service tile text */
+    --grey-light: #F3F6F9 /* projects background */
+}
+
+/* ------------------------------- 2. Typography ---------------------------*/
+html {
+    font-size: 14px;
+    color: var(--black);
+    font-family: "Roboto", serif;
+    font-weight: 300;
+}
+
+body {
+    margin: 0;
+}
+
+h1 {
+    font-family: "Merriweather", serif;
+    font-weight: 400;
+    text-align: center;
+    font-size: 2.488rem;
+}
+
+.introduction-content h2 {
+    font-family: "Merriweather", serif;
+    text-align: center;
+    font-weight: 500;
+    color: var(--purple);
+    z-index: 1;
+}
+
+.articles h2 {
+    font-family: "Merriweather", serif;
+    font-size: 2.074rem;
+    font-weight: 600;
+}
+
+header {
+    color: white;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+}
+
+.header-subtext {
+    font-size: 1.2rem;
+    text-align: center;
+}
+
+.header-button {
+    font-weight: 100;
+    font-size: 0.694rem;
+    letter-spacing: 1px;
+}
+
+.introduction p {
+    font-weight: 400;
+    text-align: center;
+}
+
+.introduction a {
+    color: var(--purple);
+}
+
+.articles p {
+    font-weight: 400;
+    font-size: 1rem;
+}
+
+.articles a {
+    font-weight: 600;
+}
+
+.squiggle-line {
+    width: 3.333rem;
+    margin-top: -1.111rem;
+}
+
+.project-link {
+    color: var(--purple);
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 0.833rem;
+}
+
+.service h3 {
```

Comment:
Nergens anders wordt een h3 gebruikt behalve bij service. h3 zou dus voldoende zijn geweest.

Probeer in de typography alleen de globale elementen op te pakken, dus echt de h3,4,5 of de p elementen.